tests/test_controller_errors.py

```diff
@@ -202,11 +196,11 @@ def test_starting_entity(test_dir, wlmutils, entity, entity_2):
     step_settings = RunSettings("echo")
     step = MockStep("mock-step", test_dir, step_settings)
     test_launcher = wlmutils.get_test_launcher()
-    if test_launcher == "dragon":
-        step = DragonStep("mock-step", test_dir, step_settings)
+    # if test_launcher == "dragon":
```

Comment:
don't forget to delete your commented code

Reply:
Done